### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by edenh**
- Landing page sketch
- Move docs to `/docs`
- Some docs cleanup

landing
landing-mobile

### CODE REVIEW

File: docs/src/Demo.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+  @media only screen and (max-width: 940px) {
+    > div {
+      width: 100%;
+      padding: 20px;
+    }
+  }
+`;
+
+const HelloWorldview = () => {
```
Comment (vidaaudrey) [inline docs/src/Demo.js]: nit: HelloWorldView

File: docs/src/Docs.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+function DefaultMain({ name, ...rest }) {
+  return (
+    <div>
+      <h1>WIP</h1> Coming soon...
+    </div>
+  );
+}
+
+const Docs = (props) => {
+  console.log(props);
```
Comment (vidaaudrey) [inline docs/src/Docs.js]: nit: remove

File: docs/src/Docs.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  const [isMobileNavOpen, toggleMobileNav] = useState(0);
```
Comment (vidaaudrey) [inline docs/src/Docs.js]: nit: useState(false), since it's a boolean value

File: docs/src/App.js [hunk 4]
```diff
@@ -4,167 +4,19 @@
...
-                        path={`${route.path}${subRoute.path}`}
-                        component={componentList[subRoute.main] || DefaultMain}
-                      />
-                    </div>
-                  ))}
-                </React.Fragment>
-              ))}
-            </Main>
-          </MainContainer>
+          <Route exact path="/" component={Landing} />
```
Comment (vidaaudrey) [inline docs/src/App.js]: Switch might be a good option here, so we can catch all routes that don't have `/docs` and render the landing page. 
```
     <Switch>
            <Route path="/docs" component={Docs} />
            <Route path="/" component={Landing} />
          </Switch>
```

File: docs/src/Demo.js [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+      margin: 24px 0;
+      display: flex;
+      align-items: center;
+      font-size: 16px;
+      color: #edcc28;
+
+      input[type="range"] {
+        width: 120px;
+        margin-right: 20px;
+        -webkit-appearance: none;
```
Comment (vidaaudrey) [inline docs/src/Demo.js]: `styled-components` should handle vendor prefixing automatically

File: docs/src/Landing.js [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+const DocsFooterLink = styled(Link)`
+  display: block;
+  color: #edcc28 !important;
+  font-size: 18px;
+`;
+
+const Landing = () => (
+  <OuterContainer>
+    <InnerContainer>
```
Comment (vidaaudrey) [inline docs/src/Landing.js]: Add a redirect here to redirect to the root url:
```
   <Route exact path="/">
      {({ match }) => !match && <Redirect to="/" />}
    </Route>
```

Comment (edenh) [reply docs/src/Landing.js]: I added the redirect to the end of the switch in `App.js`:
```
<Switch>
    <Route path="/docs" component={Docs} />
    <Route exact path="/" component={Landing} />
    <Redirect to="/" />
</Switch>
```

**Pull request merged**